test/internals/reply.test.js

```diff
@@ -18,6 +18,7 @@ const {
 const fs = require('fs')
 const path = require('path')
 const warning = require('../../lib/warnings')
+const { request } = require('undici')
```

Comment:
this file uses `sget` to exercise http. Could you stick to that?

Reply:
Yes of course